extensions/wikia/ContentReview/ContentReviewImportJS.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Wikia\ContentReview;
+
+class ImportJS {
+	const IMPORT_SCRIPTS = 'ImportJS';
+	const IMPORT_SCRIPTS_FUNCTION = 'importWikiaScriptPages';
+	const IMPORT_SCRIPTS_KEY = 'content-review-importjs';
+	const IMPORT_SCRIPTS_VERSION = '1.0';
+
+	/**
+	 * Get js script to load imports
+	 *
+	 * @return String
+	 */
+	public function getImportScripts() {
+		$importScript = \WikiaDataAccess::cache(
+			$this->getImportJSMemcKey(),
+			2592000, // 30 days,
+			function() {
+				$importScript = '';
+				$title = \Title::newFromText( self::IMPORT_SCRIPTS, NS_MEDIAWIKI );
+
+				if ( $title instanceof \Title && $title->getArticleID() != 0 ) {
```

Comment:
I'd suggest changing `$title->getArticleID() != 0` to `$title->exists()`. It's the exact same check, but `exists` has more meaning to someone reading the code about what you're actually wanting to check.